backend/tournesol/views/stats.py

```diff
@@ -104,13 +118,26 @@ def get(self, request):
 
             comparisons = Comparison.objects.filter(poll=poll)
             comparison_count = comparisons.count()
+            # if len(comparisons) > 0:
+            #     print("\n\nTimezone now:", timezone.now())
+            #     print("\n\nlastWeek:", time_ago(days=daysParam, hours=hourParam, minutes=minuteParam, seconds=secondParam))
+            #     print("\n\nMondayDiff:", mondayDiff, mondayDiff.day,daysParam, hourParam, minuteParam, secondParam)
+            #     print("\n\nTimeago weekday:", timezone.now().weekday())
+            #     print("\n\nComparison", comparisons[0])
+
+            # We should optimize the following two calls so that they are called only when the server updates
+            # the count, and not when the client calls this API
             last_month_comparison_count = comparisons.filter(
                 datetime_lastedit__gte=time_ago(days=self._days_delta)
             ).count()
+            current_week_comparison_count = comparisons.filter(
+                datetime_lastedit__gte=time_ago(days=daysParam, hours=hourParam, minutes=minuteParam, seconds=secondParam)
```

Comment:
I think we can simplify this part:

```python
from datetime import datetime, time

# this is Monday 00:00:00 
last_monday = datetime.combine(time_ago(days=timezone.now().weekday()), time.min)

# then in the filter we use our datetime object directly
current_week_comparison_count = comparisons.filter(datetime_lastedit__gte=last_monday)
```

It will be less error-prone than having to manually work with days, hours, minutes and seconds.